gen/generator.jl

```diff
@@ -1,53 +0,0 @@
-using Clang
```

Comment:
Let's keep `generator.jl` Since implot is using templates, each new plot type added results in about a dozen new C functions on the cimplot side to wrap. Easier to run the script, copy the new set of functions and cleanup.

Reply:
Whatever, but the files in gen are outdated and differ from the same files in src/libcimplot
No problem if the were identical. I searched for errors and couldn't find the cause until I found the real files in src/libcimplot

Reply:
Ah of course, generator.jl deleted by mistake...

Reply:
Yes--the redundant/outdated output is fine to remove. The script is handy, though :+1:

Reply:
reverted on master